benchmark/test/reference/conversion.profile.stderr

```diff
@@ -4,170 +4,79 @@ Running on reference(0)
 Running with 0 warm iterations and 1 running iterations
 The random seed for right hand sides is 42
 The formats are coo,csr
-Benchmarking conversions. 
-Running test case
-{
-    "size": 100,
-    "stencil": "7pt",
-    "conversions": {}
-}
-Matrix is of size (125, 125)
-DEBUG: begin stencil(125,7pt)
-DEBUG: begin allocate
-DEBUG: end   allocate
-DEBUG: begin allocate
-DEBUG: end   allocate
-DEBUG: begin allocate
-DEBUG: end   allocate
+Running test case stencil(100,7pt)
+Matrix is of size (125, 125), 725
 DEBUG: begin components::aos_to_soa
 DEBUG: end   components::aos_to_soa
+DEBUG: begin stencil(100,7pt)
+	Running conversion: coo-read
+DEBUG: begin coo-read
+DEBUG: begin repetition
+DEBUG: begin copy
+DEBUG: end   copy
+DEBUG: begin copy
+DEBUG: end   copy
+DEBUG: begin copy
+DEBUG: end   copy
+DEBUG: end   repetition
+DEBUG: end   coo-read
+	Running conversion: coo-csr
 DEBUG: begin coo-csr
-DEBUG: begin allocate
-DEBUG: end   allocate
+DEBUG: begin copy
+DEBUG: end   copy
+DEBUG: begin copy
+DEBUG: end   copy
+DEBUG: begin copy
+DEBUG: end   copy
 DEBUG: begin components::fill_array
 DEBUG: end   components::fill_array
-DEBUG: begin allocate
-DEBUG: end   allocate
-DEBUG: begin allocate
-DEBUG: end   allocate
-DEBUG: begin allocate
-DEBUG: end   allocate
-DEBUG: begin components::aos_to_soa
-DEBUG: end   components::aos_to_soa
-DEBUG: begin allocate
-DEBUG: end   allocate
-DEBUG: begin free
-DEBUG: end   free
-DEBUG: begin components::convert_idxs_to_ptrs
-DEBUG: end   components::convert_idxs_to_ptrs
-DEBUG: begin free
-DEBUG: end   free
+DEBUG: begin repetition
```

Comment:
Does the `repetition` event contain all the previous issued events, or did the rewrite actually remove some unnecessary computation?

Reply:
This is actually because not everything is annotated anymore, just the repetitions (and warmup)